simapp/app.go

```diff
@@ -57,7 +57,7 @@ var (
 	)
 
 	// module account permissions
-	maccPerms = map[string][]string{
+	MaccPerms = map[string][]string{
```

Comment:
I don't see the need of making this public as it's only used on this file

Reply:
if an external module want to have their own module account's they need to recreate the supply keeper with their added module accounts. It can be useful to reuse the existing mapping rather than duplicating it with some extra additions. I could keep this private and add a getter function?

Reply:
true true. It's fine as is as it's not internally part of the `SimApp`

Reply:
Since this map is now public, it can be modified; can be dangerous. I'd rather see it private and expose a getter that dups it.

Reply:
Since this map is now public, it can be modified; can be dangerous. I'd rather see it private and expose a getter that dups it.